examples/mobile/UIComponents/components/dialogs/dialog.js

```diff
@@ -11,12 +16,31 @@
 		for (idx = 0; idx < buttons.length; idx++) {
 			buttons[idx].addEventListener("vclick", onClick);
 		}
+		progressPage.addEventListener("popupshow", onProgressPageShow);
 	}
 
 	function onPageHide() {
 		for (idx = 0; idx < buttons.length; idx++) {
 			buttons[idx].removeEventListener("vclick", onClick);
 		}
+		progressPage.addEventListener("popuphide", onProgressPageHide);
+	}
+
+	function onProgressPageShow() {
+		var progressbarWidget = new tau.widget.Progress(progressbar);
+
+		setTimer = setTimeout(function () {
+			progressbarWidget.value(100);
+		}, 1000);
+
+		getTimer = setTimeout(function () {
+			progressPercent.innerHTML = "100%";
```

Comment:
Wouldn't it be a getTimer to get the value using get?
```js
progressPercent.innerHTML = progressbarWidget.value;
```

Reply:
done.